### DISCUSSION TITLE
Backup Jellyfin data on Windows.

### DISCUSSION THREAD

**Discussion by fernvenue** [jellyfin/jellyfin#10032] [Q&A] [answered]
Hi, I'm using Jellyfin Server on Windows, and I wanna backup my Jellyfin data, could anyone please tell me what should I do? Currently, I just copied `C:\ProgramData\Jellyfin` folder, is that enough? By the way, all my media data stored on my NAS, so I just need to backup Jellyfin configuration and user data.

**Comment by gummykage** [comment, marked as answer]
You need to figure out where jellyfin is installed and where your config folder is. These two main default locations you only need to back up the one which contains the config (cache, data, log, etc.). This folder can be used later to restore any installation, moved to other OS even, and even make a standalone where you do a jellyfin.exe -d "your config folder" and do a standalone server without having to actually be installed.

**Reply by fernvenue** [reply]
Hi @gummykage, thanks for your reply, it works for me, and actually just backup the data folder is enough, for me it's in `C:\ProgramData\Jellyfin\Server` :)
